`Marketing` > `Marketing & Content` handbook updates (#6010)
PostHog/posthog.com: contents/handbook/growth/marketing/index.md

showTitle: true
---

Marketing & Content exists to drive organic, word-of-mouth growth by:

1. Producing world-class top-of-funnel content for our ICP

> 2. Ensuring users can autonomously find answers to their questions

Author: ```suggestion
2. Ensuring the information users need is available, high-quality and easy to find
```

> 

Author: This the only major change. Thoughts on this framing?

Reviewer: Pedantic of me, but: 'can autonomously find' feels like it's a discoverability objective and thus may fall more to website and docs, vs something more content focused like 'ensuring there are answers to users' questions'. 

This is pedantic and I don't actually have a strong feeling here (mainly because I'd personally just work across both areas if I was able). But this is further reflected in a later edit where you reiterate this point and attach it to W+D there: 

```
- **Documentation** Working with Website & Docs, the content team ensures docs are maintained to a high standard, and users can easily find answers to their questions before resorting to support.
```

This is also ties in slightly to the conversation in Slack where it was suggested an individual marketing owner should own their respective area of the docs. We're due to discuss that in Sprint meeting, but it strikes me there's a philosophical shift there: viewing the docs as a marketing responsibility, vs as an extension of the product. 

Again, no super strong feelings here but I'm curious if this matches to your intention.

Reviewer: Yeah, I guess I'd just say that an obvious way to solve `2. Ensuring users can autonomously find answers to their questions` as an objective would be to improve the site search further, rebuild the docs structure, change the way the API docs are built, etc. 

I'd suggest changing this to make it clear it's owned jointly, or rephrase. 

```suggestion
Marketing & Content exists to drive organic, word-of-mouth growth by:

1. Producing world-class top-of-funnel content for our ICP
2. Work with Website and Docs to ensure the information users need is available, high-quality and can be easily found 

```

Author: For elegance, I'll drop the "Work with Website and Docs" bit. It's included further down and we know it's a shared objective.

- **Speed:** We want to be highly reactive, low process, and reliant on other teams as little as possible to ship things. We want to get stuff wrong quickly, then iterate. 

- **Content for our ideal customer:** We deliver genuinely useful insights about things those customers care about (can be purely product-related, but also general advice). We're pitching our content at ‘self-servers’. Ahrefs and HubSpot are examples of companies who do this brilliantly for their audience.

> - **Documentation** Working with Website & Docs, the content team ensures docs are maintained to a high standard, and users can easily find answers to their questions before resorting to support.

Reviewer 2: 🙌🏼

> - **Documentation** Working with Website & Docs, the content team ensures docs are maintained to a high standard, and users can easily find answers to their questions before resorting to support.

Author: ```suggestion
- **Documentation** We work with Website & Docs, and product teams, to maintain docs to a high standard, and ensure users can easily find answers to their questions
```

Author: Tweaked this reflect convos elsewhere
